crates/uv-python/src/interpreter.rs

```diff
@@ -204,15 +204,17 @@ impl Interpreter {
 
     /// Returns the [`Tags`] for this Python executable.
     pub fn tags(&self) -> Result<&Tags, TagsError> {
-        self.tags.get_or_try_init(|| {
```

Comment:
This isn't stable, unfortunately, so it's manually implemented below.

Reply:
Some day https://github.com/rust-lang/rust/issues/109737 🙏